### ISSUE TITLE
Make it only scrollable vertically

### ISSUE THREAD

**Issue by jeelpatel17** [jeelpatel17/jeelpatel17#1]
# Can anyone help me out with this!
The portfolio one-page site is also scrolling horizontally, instead of scrolling vertically only. There's only one thing to implement but I don't know which.
# Contribute
It's a static site, so in order to contribute, you can directly clone into your machine.
```Git Bash / CMD
git clone https://github.com/jeelpatel17/jeelpatel17
git add .
git commit -m "Keep it short and simple, describe the function the code will do, not the code itself"
git push```

**Comment by Shivangi-ch** [comment]
Sir ,can i try to fix the bug?

**Comment by riandy-dimas** [comment]
Hi! I have tried to fix this issue, you can see my PR here #3

**Comment by Shivangi-ch** [comment]
@jeelpatel17 please review my PR here #4
